### ISSUE TITLE
adonis4 disable csrf protection

### ISSUE THREAD

**Issue by protzi** [adonisjs/adonis-framework#645]
According to doc, this should disable csrf, but it works anyway

```
  csrf: {
    enable: false,
    methods: ['POST', 'PUT', 'DELETE'],
    filterUris: [],
    cookieOptions: {
      httpOnly: false,
      sameSite: true,
      path: '/',
      maxAge: 7200
    }
  }
```

**Comment by thetutlage** [comment]
Yes, it's a bug. Will push the fix soon

**Comment by thetutlage** [comment]
Fixed and release as `@adonisjs/shield:1.0.3`

**Comment by nobuttsseb** [comment]
enable: false results in 500 for me:
    `E_INVALID_MAKE_STRING: Ioc.makeFunc expects a string in module.method format instead received App/Controllers/Http/Api/v1/SessionsController`
